src/main/java/com/vention/delivvacoreservice/controller/OrderController.java

```diff
@@ -71,4 +74,18 @@ public ResponseEntity<Void> rejectTheOrder(@RequestBody @Valid OrderOfferDTO dto
     public ResponseEntity<List<OrderResponseDTO>> getOrderList() {
         return ResponseEntity.ok(orderService.getOrderList());
     }
+
+    @GetMapping("/order-filter")
```

Comment:
Think we can delete /order-filter. It's like syntactic sugar.

GET /api/v1/orders will be enought